docs/modules/datasources/pages/performance_datasource.adoc

```diff
@@ -176,27 +186,40 @@ An optional third argument specifies an exact resource-type name (e.g. `interfac
 Regular expressions in the template query can then be used to filter which resource IDs are used.
 
 To query a specific node by ID, you can use an expression similar to:
-```
-nodeResources(123)
-```
+
+ nodeResources(123)
 
 To query a specific node by foreign source and foreign id, you can use an expression similar to:
-```
-nodeResources(FS:FID)
-```
+
+ nodeResources(FS:FID)
 
 To query only SNMP interface resources on the node with ID 42, and to display the interfaces resources' labels rather than their often-cryptic IDs, you might use:
-```
-nodeResources(42, label, interfaceSnmp)
-```
+
+ nodeResources(42, label, interfaceSnmp)
 
 The available arguments are as follows:
 [options="header, %autowidth"]
+[cols="1,1,1,3"]
 |===
-| Name              | Required | Default  | Description
-| resourceId        | Required | `(none)` | The ID of the node (either databaseId or foreignSource:foreignId) for which to display resources.
-| textProperty      | Optional | `id`     | One of `id`, `label`, or `name` to display alternate string values in the variable drop-down menu.
-| resourceType      | Optional | `*`      | Resource type filter to limit the types of resources returned.
+| Name
+| Required
+| Default
+| Description
+
+| resourceId
+| Required
+| none
+| The ID of the node (either databaseId or foreignSource:foreignId) for which to display resources.
+
+| textProperty
+| Optional
+| id
+| One of `id`, `label`, or `name` to display alternate string values in the variable drop-down menu.
+
+| resourceType
+| Optional
+| *
```

Comment:
To confirm ... is the asterisk the default, or is this to indicate that there is no default?

Reply:
@aramosvizcarra, can you confirm whether the asterisk is the default value?